table/queries/merged_reduced_scans.sql

```diff
@@ -12,6 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+CREATE TEMP FUNCTION AddOutcomeEmoji(outcome STRING) AS (
+  CASE
+    WHEN STARTS_WITH(outcome, "setup/") THEN outcome
```

Comment:
We were using ❔ for invalid tests. I think we should have prefixes for all of them. This was it's easier to say that those outcomes are ignored for the expected rate.

Reply:
added prefixes for all